### PR TITLE
Use :option: directive in --directory help text

### PR DESCRIPTION
**Pull request by victorlin**
## Description of proposed changes

Improves rendering of reStructuredText.

## Related issue(s)

N/A

## Checklist

- [x] Checks pass
- [x] [Preview](https://nextstrain--401.org.readthedocs.build/projects/cli/en/401/commands/build/#cmdoption-nextstrain-build-arg-directory) looks good

### CODE REVIEW

File: nextstrain/cli/command/build.py [hunk 1]
```diff
@@ -199,7 +199,7 @@ def register_parser(subparser):
     parser.add_argument(
         "directory",
         help    = "Path to pathogen build directory.  "
-                  "Required, except when the AWS Batch runtime is in use and --attach and either --no-download or --cancel are given.  "
+                  "Required, except when the AWS Batch runtime is in use and :option:`--attach` and either :option:`--no-download` or :option:`--cancel` are given.  "
```
Comment (victorlin) [inline nextstrain/cli/command/build.py:202]: This works locally

image

but not on the [preview](https://nextstrain--401.org.readthedocs.build/projects/cli/en/401/commands/build/#cmdoption-nextstrain-build-arg-directory). Not sure why.

Comment (tsibley) [reply nextstrain/cli/command/build.py:202]: You need to run `./devel/generate-command-doc` and include its changes in the commit too. (This is also why the CI tests are failing; they're asserting that regenerating docs produces no change and that assertion is failing.)

Comment (tsibley) [reply nextstrain/cli/command/build.py:202]: Locally, the doc build arranges to regenerate automatically, so you probably have uncommitted local changes.

See also

https://github.com/nextstrain/cli/blob/878938cd1fe908e99246ea867198d95049b5ed23/doc/development.md#L190-L202

Comment (victorlin) [reply nextstrain/cli/command/build.py]: Oh, thanks! Fixed in [force-push](https://github.com/nextstrain/cli/compare/70a217a6351668a806dbf7e2323f19ac0b601d60..614eb34e09773d8d09596c1e664dde1a57569166).

**Review by tsibley — APPROVED** [review]

**Review by genehack — APPROVED** [review]

**Pull request merged**